.github/workflows/build.yaml

```diff
@@ -51,7 +39,10 @@ jobs:
           path: bin
 
       - name: Run lint checks
-        run: make lint
+        uses: golangci/golangci-lint-action@v6
+        with:
+          version: v1.59.1
```

Comment:
Currently we are using higher version
https://github.com/stackrox/kube-linter/blob/0c713814a586e4f587f1f04335bfe28ce0847265/tool-imports/go.mod#L6